### ISSUE TITLE
Are you able to train the model?

### ISSUE THREAD

**Issue by weedwind** [JusperLee/DANet-For-Speech-Separation#1]
Hi, you mentioned that the loss did not decrease. Just wondering have you figured out the problem?

**Comment by JusperLee** [comment]
Hello, there is no solution to the problem of loss failing to decrease.  I personally feel that there may be some correlation with the loss they proposed, because the model code and the loss code are written by the author.

**Comment by weedwind** [comment]
> Hello, there is no solution to the problem of loss failing to decrease. I personally feel that there may be some correlation with the loss they proposed, because the model code and the loss code are written by the author.

I guess you can try reducing your learning rate.

**Comment by JusperLee** [comment]
@weedwind I have tried to reduce the learning rate, and I have also tried to change the type of optimizer.  It is speculated that there may be a problem with the loss calculation.  The data preprocessing part should be fine.

**Comment by JusperLee** [comment]
you can check the code with https://github.com/naplab/DANet
